src/ralph/lib/mixins/models.py

```diff
@@ -1,10 +1,47 @@
 # -*- coding: utf-8 -*-
+from django.contrib.contenttypes.models import ContentType
 from django.core.urlresolvers import reverse
 from django.db import models
 from django.utils import timezone
 from django.utils.translation import ugettext_lazy as _
 
 
+def get_admin_url(obj, action):
+    content_type = ContentType.objects.get_for_model(obj.__class__)
```

Comment:
couldn't you get this info directly from obj/model instead of using ContentType?